### PR TITLE
Adding support for (get-model) command

### PR DESCRIPTION
**Pull request by blishko**

### CODE REVIEW

**Comment by blishko** [comment]
In my application I needed to make sure that ddSMT does not eliminate the (get-model) command during simplification, hence the change in `_substitute_cmds` method. I am not sure if there is a better way to do that or if it interacts with other ways of using ddSMT.

**Comment by mpreiner** [comment]
You can use a wrapper script to check the existence of a (get-model) command in the input. I'd not prohibit ddSMT from eliminating (get-model) in general.

**Comment by blishko** [comment]
>You can use a wrapper script to check the existence of a (get-model) command in the input.

That's a good point. Thanks!

I can remove the commit. What do you prefer revert or rewriting with push force?

**Comment by mpreiner** [comment]
Both is fine, we'll squash and merge the PR anyways :-)

File: parser/ddsmtparser.py [hunk 1]
```diff
@@ -1342,6 +1344,9 @@ def is_exit (self):
     def is_setlogic (self):
         return self.kind == KIND_SETLOGIC
 
+    def is_getmodel (self):
+        return self.kind == KIND_GETMODEL
+
```
Comment (mpreiner) [inline parser/ddsmtparser.py:1349]: This method shouldn't be needed anymore.

**Pull request merged**